### ISSUE TITLE
Deciding on a branching strategy?

### ISSUE THREAD

**Issue by VagishVela** [Neo11/ChitChat#29]
I think we should commit to the "develop" branch, then when we do a release we merge to master then do the release off there. This allows us to have tested code in "master" and development code in "develop".

Also how do you think we should do testing? Do we all test a release before we publish it?

**Comment by kapitainsky** [comment]
IMHO yes - it should be tested. What about to create “release candidate”?

**Comment by c-o-m-m-a-n-d-e-r** [comment]
yes, i created develop branch for this ... first commit there please

**Comment by VagishVela** [comment]
@c-o-m-m-a-n-d-e-r I've merged master into development so it should be up to date now.

**Comment by c-o-m-m-a-n-d-e-r** [comment]
thx...

**Comment by VagishVela** [comment]
As @kapitainsky suggested I think we should do release candidates. Also maybe do a monthly release (unless we are doing a rapid hotfix for a bug)?
